dev-manuals/etc/glossary.md

```diff
@@ -9,135 +9,210 @@ Imagine the phrase: `A xy is ...` when writing an explanation.
 ✅ Overworld: a world that ...
 ```
 
-[A](#a) [B](#b) [C](#c) [D](#d) [E](#e) [F](#f) [G](#g) [H](#h) [I](#i) [J](#j) [K](#k) [L](#l) [M](#m) [N](#n) [O](#o) [P](#p) [Q](#q) [R](#r) [S](#s) [T](#t) [U](#u) [V](#v) [W](#w) [X](#x) [Y](#y) [Z](#z)  
+[A](a) [B](b) [C](c) [D](d) [E](e) [F](f) [G](g) [H](h) [I](i) [J](j) [K](k) [L](l) [M](m) [N](n) [O](o) [P](p) [Q](q) [R](r) [S](s) [T](t) [U](u) [V](v) [W](w) [X](x) [Y](y) [Z](z)
+
```

Comment:
this change doesnt work, you cant click the Letters anymore

Reply:
Not on GitHub, but on ReadTheDocs.

Reply:
And I'm in favor of having working links on ReadTheDocs as they tend to be more user friendly.

Reply:
So which of the two do we choose then?